pages/1.14/administering-clusters/draining-a-node/index.md

```diff
@@ -0,0 +1,77 @@
+
+### Maximum Grace Period
+
+It is possible to set a max grace period when draining, which sets an upper bound on the duration of the kill grace period. When this is set, the minimum of the task's grace period and the max grace period will be used when killing the task.
+
+### Decommissioning an Agent After Draining
+
+In some cases, such as autoscaled cloud instances which are later scaled down, you may know that after the node is drained and taken down, it will never come back again. Each agent node is uniquely identified by metadata on its hard disk, so even if other instances are scaled up in the future, if they don't retain the persisted metadata of the previous instances, they are viewed as new nodes by the DC/OS cluster.
+
+In cases where the unique agent will not return after it is drained and removed from the cluster, you can specify an option which will automatically decommission the node after it has finished draining, which removes the agent's metadata from the cluster. If this option is not specified, the cluster will remember the agent node in case it reregisters with the cluster later on.
+
+## Draining with the CLI
+
+To drain a node with the DC/OS CLI, use the `dcos node drain` command:
+
+    ```bash
+    dcos node drain <mesos-id>
+    ```
+
+The maximum grace period and/or decommission options may be specified like so:
+
+    ```bash
+    dcos node drain <mesos-id> --timeout=10m --decommission
+    ```
+
+Once draining on an agent has begun, you can monitor the tasks on that node by running:
+
+    ```bash
+    dcos task list --agent-id <mesos-id>
+    ```
+
+Overall progress of draining on the node may be monitored with the following command:
+
+    ```bash
+    dcos node list
+    ```
+
+Locate the relevant agent node in the list and see its `STATUS` field. The usual status of a node will be `ACTIVE`. When draining is initiated on the node, it will transition to `DRAINING`. Once all tasks have been killed and their terminal status updates have been acknowledged by the relevant services, and once all resource operations on the node have finished, it will transition to `DRAINED`. When the node is in the `DRAINED` state, draining is complete.
+
+If the `--decommission` option was included in the initial drain command, the node's metadata will be automatically removed from the cluster when draining is complete. Otherwise, the cluster will wait for the node to be manually reactivated when any planned maintenance or other update procedures are finished. Before performing maintenance, stop the `dcos-mesos-agent` systemd unit on the node so that the agent no longer attempts to register with the master. When maintenance is complete and you are ready to reactivate the node and allow it to rejoin the cluster, you may do so with the following command:
+
+    ```bash
+    dcos node reactivate <mesos-id>
+    ```
+
+Then start the `dcos-mesos-agent` systemd unit on the node and the agent will rejoin the cluster.
```

Comment:
it's probably useful to just write down the sysctl command to start the agent so that users can copy paste.